server/interfaces/events.ts

```diff
@@ -5,9 +5,21 @@ export interface Event {
   date: string
   records: Record[]
 }
-
 export interface Events {
   backUrl: string
   events: Event[]
   heading?: string
 }
+
```

Comment:
I created new interfaces here as the visit details page requires an address field in the timeline. The other pages that use a timeline don't